test/src/unit-cppapi-update-queries.cc

```diff
@@ -295,8 +295,6 @@ TEST_CASE_METHOD(
   std::vector<sm::UpdateValue> uvs2;
   val = 2;
   uvs2.emplace_back("a1", &val, sizeof(int32_t));
-  val = 1;
```

Comment:
I think the goal of this test was to add two update values to an update condition. Let's add an attribute to the schema so we can add a second update value to this condition.

Reply:
Added a second attribute here.